apps/api/R/submit.workflow.R

```diff
@@ -45,6 +45,17 @@ submit.workflow.list <- function(workflowList, userDetails) {
   )
   )
   
+  if(is.null(workflowList$model$id)) {
```

Comment:
Could you add a comment explaining what you're trying to do at this step? To me it looked like you were using the model type and revision to look up the model_id if that ID was missing, but instead you're just putting that same information back into the settings, but not adding the ID.

Reply:
Thanks for this...I had something completely different in mind, and I ended up doing something different...will rectify this.

Reply:
![image](https://user-images.githubusercontent.com/31898274/89425966-3e906b00-d757-11ea-895f-665139505130.png)

This is something I wanted to do...coz in some xml files, they have `model_id` instead of `model_name` & `revision`, & `model_name` & `revision` are required to form the name of the rabbitmq queue...that is why I wanted to store these values if they were not provided.

Reply:
That makes more sense. But I'd still say it would be good to put a comment before code blocks like these to explain the what/why of a step (the "how" should be clear from the code). That makes it a lot easier to read and for others to debug / build-upon.